clients/node/client-rds-data-node/protocol/AwsRestJson1_1.ts

```diff
@@ -16,7 +16,7 @@ import { ResponseMetadata, Endpoint } from "@aws-sdk/types";
 
 export function executeStatementAwsRestJson1_1Serialize(
   input: ExecuteStatementRequest,
-  utils: SerializerUtils & { endpoint: Endpoint }
+  utils: SerializerUtils
```

Comment:
Endpoint can be dropped from the import above.  

ResponseMetadata can also be rolled into the SerializerUtils import as well.

Reply:
Yea. I need to rename `SerializerUtils` to `SerializerContext` as well.

> ResponseMetadata can also be rolled into the SerializerUtils import as well

How would it look like?

Reply:
```
import { SerializerUtils, DeserializerUtils, ResponseMetadata } from "@aws-sdk/types";
```

I just meant to combine the imports from /types